components/TeamOverview.vue

```diff
@@ -11,11 +11,14 @@
       @keyup.ö="addPoints(3,-1)"
     />
     <ol v-if="gamepadsWithButtons.length" class="flex justify-around">
-      <li v-for="(gamepad) in gamepadsWithButtons" :class="[teamClass[gamepad.index], isButtonAPushed(gamepad) ? 'bg-green-500' : '' ]">
+      <li v-for="(gamepad) in gamepadsWithButtons" v-if="gamepad != null" :class="[teamClass[gamepad.index], isButtonAPushed(gamepad) ? 'bg-green-500' : '' ]">
```

Comment:
```suggestion
      <li v-for="(gamepad) in gamepadsWithButtons" v-if="gamepad !== null" :class="[teamClass[gamepad.index], isButtonAPushed(gamepad) ? 'bg-green-500' : '' ]">
```

The linter should throw a warning here due to the missing strict check

Reply:
I've just checked the output on my machine - I'm afraid there's no linter warning, neither here nor in Quiz.vue. In principle it works though, as it's very eager in *.js files 😄 Looking at package.json eslint seems to be configured to check .vue files as well. So I wonder why it doesn't work in my case 🤔

Reply:
Added your changes to d922bf1 - thanks!

Reply:
Strange 🤷🏻‍♂️ Could be possible that the vue plugin doesn't check expressions in v-if or so :D